src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -76,6 +76,274 @@ curie_map:
   # SNOMEDCT_2010_1_31: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/SNOMEDCT_2010_1_31/"
   OMIA: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/OMIA/"
 
+extended_prefix_map:
```

Comment:
## Add EPM
I made a mistake when recently working on #394. I added in this `extended_prefix_map` into the copy of `mondo.sssom.config.yml` in the `mondo-ingest` repo. However, the source of this file actually comes from `mondo.  So really the workflow needs to be:

Update the file in `mondo` -> pull update into `mondo-ingest`

Future considerations:
- I think that `mondo-ingest` should add this file to the `.gitignore` so that this doesn't automatically happen.
- We need to figure out how to make sure that when files like this get updated in `mondo`, `mondo-ingest` can automatically update. This happens automatically in fresh builds, but not in development.

Reply:
I think these two files (mondo and mondo ingest mapping configs) may have to diverge (separately managed). They must be very different from one another by now.. How different are they? They serve also quite different purposes. I would tend to remove the update mechanism and curate them both separately.

Reply:
Ah, I see. Well then, let's keep them separate. Are you OK with this plan:
- [ ] 1. I use me and Charlie's new function to convert the `mondo` `curie_map` into an `extended_prefix_map` and update this PR so that `mondo.sssom.config.yml` here is internally consistent.
- [ ] 2. I update `mondo-ingest` and remove the goal for `mondo.sssom.config.yml`, and just keep it committed as a static file that needs to be maintained there.

Reply:
2 for sure. 

1 you can try, but I am not sure what the consequences of "making it consistent" will be! We shall see I guess.

Reply:
Ok, I'm going to close this, do (2) and create an issue for (1), I guess low priority.